src/main/java/org/folio/util/TokenCookieParser.java

```diff
@@ -0,0 +1,26 @@
+package org.folio.util;
+
+import org.folio.rest.impl.LoginAPI;
+
+public class TokenCookieParser {
+  public String accessToken;
+  public String refreshToken;
+
+  public TokenCookieParser(String cookieHeader) {
+    accessToken = getCookieValue(cookieHeader, LoginAPI.ACCESS_TOKEN);
+    refreshToken = getCookieValue(cookieHeader, LoginAPI.REFRESH_TOKEN);
+  }
+
+  private String getCookieValue(String cookieHeader, String key) {
+    String[] cookies = cookieHeader.split(";");
+    for (String c : cookies) {
+      String[] cookeNameValue = c.trim().split("=", 2);
```

Comment:
This trim() is not needed because the trim() in the next two lines already do this.

Reply:
Removed